File: mobile-client/android/Contract/Product/Listing/ProductCardSummary.kt
package com.smartresponsor.mobile.client.contract.product.listing

/**
 * Marketing America Corp. Oleksandr Tishchenko
 */
data class ProductCardSummary(
    val productId: String,
    val title: String,
    val subtitle: String?,
    val priceLabel: String?,
    val thumbnailUrl: String?,
    val availabilityLabel: String?,
)

File: client/ios/Contract/Product/Listing/ProductCardSummary.swift
import Foundation

// Marketing America Corp. Oleksandr Tishchenko
public struct ProductCardSummary: Sendable, Equatable {
    public let productId: String
    public let title: String
    public let subtitle: String?
    public let priceLabel: String?
    public let thumbnailUrl: String?
    public let availabilityLabel: String?
}
